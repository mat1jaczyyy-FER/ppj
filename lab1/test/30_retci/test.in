

a = 12

b = 13

// test
// test
/ /
d=15
