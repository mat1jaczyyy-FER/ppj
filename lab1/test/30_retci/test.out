IDN 3 a
OP_PRIDRUZI 3 =
BROJ 3 12
IDN 5 b
OP_PRIDRUZI 5 =
BROJ 5 13
OP_DIJELI 9 /
OP_DIJELI 9 /
IDN 10 d
OP_PRIDRUZI 10 =
BROJ 10 15
